third_party/gif_decoder/src/main/java/com/bumptech/glide/gifdecoder/GifDecoder.java

```diff
@@ -352,6 +352,7 @@ public void clear() {
             bitmapProvider.release(previousImage);
         }
         previousImage = null;
+        rawData = null;
```

Comment:
His branch is 1 PR behind, should the merge handle this?
Though it's probably better if @frindly rebases.